tests/coprocessor/test_select.rs

```diff
@@ -266,6 +267,7 @@ fn test_select() {
 
 #[test]
 fn test_group_by() {
+    init_log();
```

Comment:
Are there any function like SetUp ? so we don't need to call init_log() again and again.